fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java

```diff
@@ -103,8 +103,11 @@ public final class FeMetaVersion {
 
     public static final int VERSION_140 = 140;
 
+    // For backup global info
+    public static final int VERSION_141 = 141;
+
     // note: when increment meta version, should assign the latest version to VERSION_CURRENT
-    public static final int VERSION_CURRENT = VERSION_140;
+    public static final int VERSION_CURRENT = VERSION_141;
```

Comment:
changed fe meta version， it is very dangerous

Reply:
BackupMeta has changed,  upgrade the fe meta verison to  compatible with old versions.